R/fill.R

```diff
@@ -16,20 +16,23 @@ NULL
 #'   with `x:z`, exclude `y` with `-y`. For more selection options, see the
 #'   [dplyr::select()] documentation.
 #' @param .direction Direction in which to fill missing values. Currently
-#'   either "down" (the default) or "up".
+#'   either "down" (the default), "up", "downup" (i.e. first down and then up)
+#'   or "updown" (first up and then down).
 #' @export
 #' @examples
 #' df <- data.frame(Month = 1:12, Year = c(2000, rep(NA, 11)))
 #' df %>% fill(Year)
-fill <- function(data, ..., .direction = c("down", "up")) {
+fill <- function(data, ..., .direction = c("down", "up", "downup", "updown")) {
   UseMethod("fill")
 }
 #' @export
-fill.data.frame <- function(data, ..., .direction = c("down", "up")) {
+fill.data.frame <- function(data, ..., .direction = c("down", "up", "downup", "updown")) {
   fill_cols <- unname(tidyselect::vars_select(names(data), ...))
 
   .direction <- match.arg(.direction)
-  fillVector <- switch(.direction, down = fillDown, up = fillUp)
+  fillVector <- switch(.direction, down = fillDown, up = fillUp,
```

Comment:
Can you please format so each named argument is on its own line? http://style.tidyverse.org/syntax.html#long-lines